bec_lib/bec_lib/config_helper.py

```diff
@@ -33,6 +34,23 @@
 logger = bec_logger.logger
 
 
+@dataclass(frozen=True)
+class _ConfigConstants:
+    NON_UPDATABLE = ("name", "deviceClass", "deviceConfig", "description")
```

Comment:
* deviceConfig was updatable before (just handled before one would even get to this check)
* description should be updatable

Reply:
fixed